src/mod/thr/thr.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * thr.h
+ *
+ *  Created on: 01.03.2022
+ *      Copy Paste by: Jevgeni
+ */
+
+//
+#ifndef MOD_TIM_CLIMB_THR_H_
+#define MOD_TIM_CLIMB_THR_H_
+
+#include <chip.h>
+
+// init data needed. Choose UARt and 2 GPIO Pins to be used
+typedef struct {
+	LPC_USART_T 	*pUart;			// default is 9600baud
+	const PINMUX_GRP_T2   *pEnablePin;	// Output LPC -> GPS enable internal Voltage regulator -> we disable because we use SP Power
```

Comment:
remove this old io definiotions (copied over from GPS)

Reply:
Fixed

https://github.com/RobertK66/obc_1769_core/pull/34/commits/536712ccc92806a2e5c117fd2d5c82f9ab6059e1